hibernate-core/src/main/java/org/hibernate/dialect/Oracle12cDialect.java

```diff
@@ -62,4 +67,9 @@ public String getNativeIdentifierGeneratorStrategy() {
 	public IdentityColumnSupport getIdentityColumnSupport() {
 		return new Oracle12cIdentityColumnSupport();
 	}
+
+	@Override
+	public LimitHandler getLimitHandler() {
+		return LIMIT_HANDLER;
```

Comment:
Seems 'LIMIT_HANDLER' is only used above once. Why not use 'Oracle12LimitHndler.INSTANCE' directly without an intermediary?

Reply:
this method is used by `Loader` and `AbstractLoadPlanBasedLoader`

```
final LimitHandler limitHandler = getFactory().getDialect().getLimitHandler();
```

Reply:
My question was not about the public method, but about its internal implementation. Seems we can change its impl to:
```
@Override
public LimitHandler getLimitHandler() {
    return Oracle12LimitHandler.INSTANCE;
}

```
and we can delete the private static variable. It is just a little bit confusing about its necessity.
Feel free to ignore my comment though. Not a big deal.

Reply:
sorry I haven't understood, I'm going to change it, thanks